pkg/cli/options.go

```diff
@@ -181,13 +181,18 @@ func isHostSupported(host string) bool {
 	return false
 }
 
-// getPluginsRoot detects the host system and gets the plugins root based on the host.
+// getPluginsRoot gets the plugin root path.
 func getPluginsRoot(host string) (pluginsRoot string, err error) {
 	if !isHostSupported(host) {
 		// freebsd, openbsd, windows...
 		return "", fmt.Errorf("host not supported: %v", host)
 	}
 
+	// if user provides specific path, return
+	if pluginsPath := os.Getenv("KB_PLUGINS_PATH"); pluginsPath != "" {
+		return pluginsPath, nil
```

Comment:
Might be worth adding a check to validate that the path actually exists unless we are already doing that in the function that calls `getPluginsRoot()`.